Priyaa/HTML Basics Remake/2.Table Remake/POKEMON.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Assignment - Table Remake</title>
+	</head>
+	<body>
+		<h1>Pokemon Chart</h1>
+		<table border="1">
+			<thead>
+				<th>Image</th>
+				<th>Name</th>
+				<th>Type</th>
+				<th>Evolves Into</th>
+			</thead>
+
+			<tbody>
+				<tr>
+					<td>
+						<img
+							src="http://img4.wikia.nocookie.net/__cb20140328190757/pokemon/images/thumb/2/21/001Bulbasaur.png/200px-001Bulbasaur.png"/>
+					</td>
+					<td>Bulbasaur</td>
+					<td>Grass/Poison</td>
+					<td>
+						<a href="http://pokemon.wikia.com/wiki/Ivysaur ">Ivyasaur</a>
+					</td>
+				</tr>
+				<tr>
+					<td>
+						<img
+							src="http://img4.wikia.nocookie.net/__cb20140724195345/pokemon/images/thumb/7/73/004Charmander.png/200px-004Charmander.png"/>
+					</td>
+					<td>Charmander</td>
+					<td>Fire</td>
+					<td>
+						<a href="http://pokemon.wikia.com/wiki/Charmeleon">Charmeleon</a>
+					</td>
+				</tr>
+				<tr>
+					<td>
+						<img
+							src="http://img1.wikia.nocookie.net/__cb20140328191525/pokemon/images/thumb/3/39/007Squirtle.png/200px-007Squirtle.png"/>
+					</td>
+					<td>Squirtle</td>
+					<td>Water</td>
+					<td>
+						<a href="http://pokemon.wikia.com/wiki/Wartortle">Wartortle</a>
```

Comment:
Have a look at these:

- [Why HTTPS Matters](https://developers.google.com/web/fundamentals/security/encrypt-in-transit/why-https)

- [Why is HTTP not secure? | HTTP vs. HTTPS](https://www.cloudflare.com/learning/ssl/why-is-http-not-secure/)

- [Mixed content](https://developer.mozilla.org/en-US/docs/Web/Security/Mixed_content)

- [Deprecating Non-Secure HTTP](https://blog.mozilla.org/security/2015/04/30/deprecating-non-secure-http/)

- [What Is Mixed Content?](https://developers.google.com/web/fundamentals/security/prevent-mixed-content/what-is-mixed-content)